naucse/models.py

```diff
@@ -505,13 +660,25 @@ def __str__(self):
     url = DataProperty(info)
 
 
+class MetaInfo(Model):
+    """ Info about the current repository. """
+
+    def __str__(self):
+        return "Meta Information"
+
+    config = YamlProperty()
+
+    slug = DataProperty(config)
+    branch = DataProperty(config, default="master")
```

Comment:
Would be nice if you read this from git, and not from a config file. (But that's not blocking this PR.)